src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -46,7 +46,6 @@ public class WebScriptHostManager : ScriptHostManager
         private readonly WebJobsSdkExtensionHookProvider _bindingWebHookProvider;
 
         private bool _hostStarted = false;
-        private IDictionary<IHttpRoute, FunctionDescriptor> _httpFunctions;
```

Comment:
Now that we can store additional data with each route in the route collection, we don't need this external map